ReactiveArchitecture/Classes/nowplaying/model/result/Result.swift

```diff
@@ -11,14 +11,14 @@ import Foundation
 /**
  * Base class for results from asynchronous actions.
  */
-protocol Result : class {
+protocol Result: class {
     
     func getType() -> ResultType
     
 }
 
 enum ResultType {
-    case IN_FLIGHT
-    case SUCCESS
-    case FAILURE
+    case inFlight
+    case success
+    case failure
```

Comment:
Swift naming guidelines

Reply:
I'm trying to find where they talk about ENUMS. From ADA To C to Java...always been caps. Easier for readability that the code speaks about enums vs other types. 

Just trusting and verifying :-)